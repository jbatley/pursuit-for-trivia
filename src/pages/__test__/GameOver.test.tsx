import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter as Router } from 'react-router-dom';
import Component from '../GameOver';

describe('Page Start', () => {
  test('should render and', async () => {
    const { getByText } = render(<Router><Component /></Router>);
    expect(getByText(/game over/i)).toBeInTheDocument();
  });
  test('matches snapshot', async () => {
    const { container } = render(<Router><Component /></Router>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
