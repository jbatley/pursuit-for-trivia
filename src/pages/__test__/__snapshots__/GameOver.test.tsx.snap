// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Start matches snapshot 1`] = `
<section
  class="grid"
  style="grid-template-rows: 70% auto;"
>
  <h1
    class="h1"
  >
    GAME OVER
  </h1>
  <h1
    class="h1"
  >
    SCORE: 
  </h1>
  <div
    class="flex"
  >
    <a
      class="link"
      href="/mode-selector"
    >
      <button
        class="box"
        type="button"
      >
        Resart
      </button>
    </a>
  </div>
</section>
`;
